phoenix-coprocessors/src/main/java/org/apache/phoenix/schema/transform/Transform.java

```diff
@@ -17,19 +17,14 @@
  */
 package org.apache.phoenix.schema.transform;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import org.apache.hadoop.hbase.util.Bytes;
-import org.apache.phoenix.coprocessor.TableInfo;
+import org.apache.phoenix.scanattributes.TableInfo;
```

Comment:
Find some other package name, this is not a scan attribute.
(I know it's because I asked you to rename the package to scanatributes.)